api.md

```diff
@@ -3251,7 +3157,7 @@ SELECT get_telemetry_report(always_display_report := true);
 ```
 ---
 
-## approximate_row_count() [](approximate_row_count)
+## approximate_row_count() :community_function:  [](approximate_row_count)
```

Comment:
Isn't this in Apache-2 codebase, not TSL?  If so, it shoudn't be marked :community_function:

Reply:
yes, this is Apache. Removed.